frontend/src/pages/About.tsx

```diff
@@ -135,11 +135,19 @@ export default function About() {
                     p: <p></p>,
                     imgTeam: (
                       <img
-                        src="https://images.clothingloop.org/88ce5a09-71f6-4b03-aa10-fc3a172f7f4e/-/resize/x240/-/format/auto/-/quality/smart/Teampics.jpg"
-                        alt="faces of Paloeka and Nichon"
+                        src="https://ucarecdn.com/11253a80-a1d5-4639-a3ca-166dddcae210/PaloekaNichonlandscape.jpg"
```

Comment:
This will download the full sized version
Please send me the original, and I'll make it available on image.clothingloop.org at a reasonable size

Reply:
I just sent in in slack